src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -238,6 +274,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
       navigation.navigate('SignedChatScreen');
     }
 
+    console.log('checkpoint 7');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This console log statement seems to be a debugging artifact and should probably be removed.
```diff
-    console.log('checkpoint 7');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->